.eslintrc.json

```diff
@@ -16,6 +16,7 @@
   "rules": {
     "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
     "import/no-unresolved": "off",
-    "no-shadow": "off"
+    "no-shadow": "off",
+    "react/no-array-index-key": 0
```

Comment:
Please let's use the **correct config files**.

![image](https://user-images.githubusercontent.com/36057474/71924065-8b46a000-318e-11ea-8f37-02c188bde393.png)

See https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

If there's a reason why the config file needs to be changed, let's discuss it on this PR.